Composer/packages/server/assets/projects/ToDoBotWithLuisSample/dialogs/AddItem/AddItem.dialog

```diff
@@ -109,6 +109,6 @@
       ]
     }
   ],
-  "generator": "AddItem.lg",
-  "recognizer": "AddItem.lu"
+  "generator": "additem.lg",
```

Comment:
do we also need to rename the folder and .dialog file?

Reply:
Yes,,, git commit did not catch the file rename, now updated.